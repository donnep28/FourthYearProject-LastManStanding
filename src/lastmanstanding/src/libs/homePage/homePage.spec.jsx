import React from 'react';
import {shallow} from 'enzyme';
import HomePage from './homePage';

describe('loggedInHomePage', () => {
  describe('Snapshot', () => {
    it('renders', () => {
      const component = shallow(
          <HomePage />,
      );
      expect(component).toMatchSnapshot();
    });
  });
});
