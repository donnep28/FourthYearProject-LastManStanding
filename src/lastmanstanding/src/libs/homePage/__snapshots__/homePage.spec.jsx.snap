// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loggedInHomePage Snapshot renders 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Grid))
    container={true}
    direction="row"
    spacing={2}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={8}
      xs={12}
    >
      <PremierLeagueStandings
        results={Object {}}
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={4}
      xs={12}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
        style={
          Object {
            "marginBottom": "20px",
          }
        }
        xs={12}
      >
        <PremierLeagueFixtures
          results={Object {}}
        />
      </WithStyles(ForwardRef(Grid))>
      <WithStyles(ForwardRef(Grid))
        item={true}
        xs={12}
      >
        <PremierLeagueResults
          results={Object {}}
        />
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</Fragment>
`;
